go/vt/vttablet/tabletserver/schema/db.go

```diff
@@ -459,3 +548,12 @@ func GetFetchTableAndViewsQuery(tableNames []string, parser *sqlparser.Parser) (
 	}
 	return parsedQuery.GenerateQuery(bv, nil)
 }
+
+// GetFetchUDFsQuery gets the fetch query to retrieve all the UDFs.
+func GetFetchUDFsQuery(parser *sqlparser.Parser) (string, error) {
```

Comment:
If we are fetching only aggregate UDFs, this func should be renamed to reflect that.